### PR TITLE
created consume-events endpoint

### PR DESCRIPTION
**Pull request by anyelopetit**
### Description
Is an endpoint in charge of receiving event requests and processing them, authenticated by the header `"AppToken: {{AppToken}}"`

### CODE REVIEW

File: app/controllers/v1/events_controller.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module V1
+  class EventsController < ApplicationController
+    before_action :set_app
+    before_action :set_event_schema
+
+    #  GET /api/v1/consume-events
+    def consume
+      @event = @event_schema.events.create(
```
Comment (gbrlmrllo) [inline app/controllers/v1/events_controller.rb]: I think we should use `create!`, So we raise an exception if there is something wrong.

File: spec/controllers/v1/events_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe V1::EventsController, type: :controller do
+  context "with an invalid token" do
+    let(:token) { 'iNv4lidT0k3n' }
+    let(:app) { App.joins(:credential).find_by!(credentials: { token: token }) }
+
+    describe "GET #consume" do
```
Comment (gbrlmrllo) [inline spec/controllers/v1/events_controller_spec.rb]: GET? should be POST

File: config/routes.rb [hunk 3]
```diff
@@ -18,6 +18,7 @@
   scope :api do
     namespace :v1 do
       get "/me", to: "users#me"
+      get "/consume-events", to: "events#consume"
```
Comment (gbrlmrllo) [inline config/routes.rb]: should be `POST`

**Review by gbrlmrllo — CHANGES_REQUESTED** [review]
Nice job @anyelopetit 

I left a comments in this first review.

File: app/controllers/v1/events_controller.rb (hunk 1 above)
Comment (anyelopetit) [reply app/controllers/v1/events_controller.rb]: Done :heavy_check_mark:

File: spec/controllers/v1/events_controller_spec.rb (hunk 2 above)
Comment (anyelopetit) [reply spec/controllers/v1/events_controller_spec.rb]: Right! Done :heavy_check_mark:

File: config/routes.rb (hunk 3 above)
Comment (anyelopetit) [reply config/routes.rb]: Done :heavy_check_mark:

**Review by gbrlmrllo — APPROVED** [review]
@anyelopetit LGTM 💯 

Good refactor! 🏅

**Pull request merged**